sdk/eventgrid/Azure.Messaging.EventGrid/assets.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we also migrate the recordings from the Event Grid functions library - https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/tests/SessionRecords/OutputBindingEndToEndTests

Reply:
Done!